pontoon/base/migrations/0131_translation_active_populate.py

```diff
@@ -0,0 +1,68 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.15 on 2018-08-30 12:36
+from __future__ import unicode_literals
+
+import datetime
+
+from django.db import migrations
+
+
+def set_active_translations(apps, schema_editor):
+    Translation = apps.get_model('base', 'Translation')
+    print(datetime.datetime.now().time())
+
+    # All approved translations are active.
+    Translation.objects.filter(approved=True).update(active=True)
+    print(datetime.datetime.now().time())
+
+    # All fuzzy translations are active.
+    # Note that this wasn't the case until this patch - if a fuzzy translation
+    # had a newer sibling suggestion, that suggestion was active.
+    # However, this is a bug, and we're fixing it here.
+    Translation.objects.filter(fuzzy=True).update(active=True)
+    print(datetime.datetime.now().time())
+
+    # The most recent unreviewed suggestion for any given combination of
+    # (locale, entity, plural_form) is active, unless it has approved or fuzzy
+    # siblings.
+    # Note that currenty this logic applies to all suggestions, even rejected.
+    # However, this is a bug, and we're fixing it here.
+    unreviewed_pks = set()
+    candidates = Translation.objects.filter(
+        approved=False,
+        fuzzy=False,
+        rejected=False,
+    )
```

Comment:
[nit] you probably could use `values_list` to return only `entity_id`, `locale` and `plural_form` (that would reduce the time required by Django in order to initialize object instances).

Reply:
Wow, that splits the time in half! I'll ammend the commit.